source/includes/_05-one-time-sync-ekyc.html.md.erb

```diff
@@ -338,12 +338,26 @@ app.post("/checks", (req, res) => {
   <tbody>
     <tr>
       <td><code>check_output</code></td>
-      <td>Object structured per <code>check_output</code> below</td>
-      <td>If check did not error</td>
+      <td>Object structured in <a href=#check_output>check_output</a> below</td>
+      <td>Only when the check does not error or there is no result</td>
```

Comment:
It's come to my attention that there are cases where you would want to provide some fields in the `check_output`, *and* the `result` field, so we should revert this line. The description for `result` is fine as-is.

We should change the `matches` field to say it's only required when `result` is not present.